Sources/ApodiniDatabase/Handlers/Create.swift

```diff
@@ -0,0 +1,30 @@
+import Fluent
+import Apodini
+
+/// A Handler that creates, if possible, an object in the database that conforms to `DatabaseModel` out of the body of the request.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+public struct Create<T: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+    
+    @Parameter
+    private var object: T
+
+    public func handle() -> T? {
+        // swiftlint:disable all
+        let result = try! object
+            .save(on: database)
+            .map { _ in
+                self.object
+            }
+            .wait()
```

Comment:
I would suggest to wait for the merging of #94 or #98 or a mix between the two ideas to we don't introduce blocking code in develop. It would also be great to get your input in the ideas as well as the #97 as your code would be one of the main users of asynchronous code.